cache/manager_test.go

```diff
@@ -662,6 +663,267 @@ func TestExtractOnMutable(t *testing.T) {
 	checkNumBlobs(ctx, t, co.cs, 0)
 }
 
+func TestLinkNydusBlob(t *testing.T) {
```

Comment:
Could we add nydus on all other compression-related tests as well (e.g. TestGetRemotes)?

Reply:
Thanks, will add it.